ArchiSteamFarm/ArchiSteamFarm.csproj

```diff
@@ -30,7 +30,6 @@
 	</ItemGroup>
 
 	<ItemGroup Condition="'$(TargetFramework)' == 'net48'">
-		<PackageReference Include="JustArchiNET.Madness" />
```

Comment:
Remove this also from other of our projects, I'm surprised it doesn't outright yell at you for duplicate package reference, seems msbuild is fine with that.

Reply:
No wait, you can't, those have compile references only, I wonder now if it works then as should, gonna check artifacts...

Reply:
It's fine, disregard this suggestion.